openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -61,6 +61,7 @@
     "alert-lowercase": "alert",
     "alert-lowercase-plural": "alerts",
     "alert-plural": "Alerts",
+    "alert-synced-successfully": "Alert synced successfully",
```

Comment:
Let's add these changes to other language files as well, with the respective translations.